client/verta/tests/test_endpoint/test_endpoint.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+import verta
+
+
+@pytest.mark.skip("endpoint not yet available in backend")
```

Comment:
I thought Endpoint is available in `dev`. Could you do me a favor and confirm this? If it's not available in dev, then this is good 👍

Reply:
it works on dev